Changelog

```diff
@@ -1,4 +1,83 @@
 Unreleased - sledgehammer999 <sledgehammer999@qbittorrent.org> - v4.5.0
+    - FEATURE: Add "Auto resize columns" functionality (Chocobo1)
+    - FEATURE: Allow to use Category paths in "Manual" mode (glassez)
+    - FEATURE: Allow to disable Automatic mode when "temp" path changed (glassez)
```

Comment:
I don't recall such a feature. I need to see what it really refers to.

Reply:
c5ff9b353c4179a2b335740f66425e621f346016

Reply:
Ok. I see it's about changing "Default incomplete path" setting.

Reply:
```suggestion
    - FEATURE: Allow to disable Automatic mode when default "temp" path changed (glassez)
```